src/app/api/attendees/count/route.ts

```diff
@@ -0,0 +1,26 @@
+import { NextResponse } from 'next/server'
+import { cookies } from 'next/headers'
+import { createServerClient } from '@/utils/supabase'
+
+export const runtime = 'edge'
+
+export async function GET(request: Request) {
+  const cookieStore = cookies()
+  const supabase = createServerClient(cookieStore)
+
+  const url = new URL(request.url)
+  const eventid = url.searchParams.get('eventid')
```

Comment:
I'd recommend an array of passed in event ID... so "eventids" and check a set.  I'd recommend say 10 to pass in so you can get a single api call responding with 10 events at a time.  Reduces the single shot hits for each event this approach currently takes.  Happy to chat if you want to talk it over.